backend/.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow runs ESLint, Prettier, unit tests, and e2e tests.
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm install
```

Comment:
You need to replace all the scripts with pnpm. Also, just go through `package.json` once since the names don't match.